internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -62,3 +64,13 @@ func criStart(t *Target, data interface{}) error {
 	_, _, err := t.ssh("systemctl", "enable", "--now", "crio")
 	return err
 }
+
+func criStop(t *Target, data interface{}) error {
+	_, _, err := t.ssh("systemctl", "stop", "crio")
+	return err
+}
+
+func criWipePods(t *Target, data interface{}) error {
+	_, _, err := t.ssh("crictl", "rmp", "-fa")
```

Comment:
This is very impactful, and is basically the core of the new behaviour we introduce here. Did we fully investigate its impact?

Reply:
It's the only solution we have at the moment. Any workload that is formed only by one pod will have downtime.